### PR TITLE
Added ota-update node

### PR DESCRIPTION
**Pull request by Dirnei**
The ota-update node can start ota updates and will send messages on: start, end and progress change.
Automatic start of updates is also available if configured.

### CODE REVIEW

File: src/ota.html [hunk 1]
```diff
@@ -122,10 +122,10 @@ <h3>Outputs</h3>
         <dd>
             Satus message of the current progress. It is one of the following messages:
             <ul>
-                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end"</li>
-                <li><code>Update succeded</code> will be sent when the update succeded (Output 1 - "start/end")</li>
+                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end")</li>
+                <li><code>Update succeded</code> will be sent when the update succeeded (Output 1 - "start/end")</li>
                 <li><code>Update failed</code> will be sent when the update failed (Output 1 - "start/end")</li>
-                <li><code>Progress changed</code> will be sent when the update succeded (Output 2 - "progress")</li>
+                <li><code>Progress changed</code> will be sent when the progress changed (Output 2 - "progress")</li>
```
Comment (Andreas-Huber) [inline src/ota.html:128]: Makes sense?

File: src/ota.html [hunk 2]
```diff
@@ -122,10 +122,10 @@ <h3>Outputs</h3>
         <dd>
             Satus message of the current progress. It is one of the following messages:
             <ul>
-                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end"</li>
-                <li><code>Update succeded</code> will be sent when the update succeded (Output 1 - "start/end")</li>
+                <li><code>Update started</code> will be sent when the update starts (Output 1 - "start/end")</li>
+                <li><code>Update succeded</code> will be sent when the update succeeded (Output 1 - "start/end")</li>
```
Comment (Andreas-Huber) [inline src/ota.html:126]: In theory, you could change this also to "succeeded" - it's not wrong, just not so common I would say.

File: src/ota.html (hunk 1 above)
Comment (Dirnei) [reply src/ota.html:128]: yep. thanks a lot!

File: src/ota.html (hunk 2 above)
Comment (Dirnei) [reply src/ota.html:126]: sorry, i dont understand what you mean

Comment (Andreas-Huber) [reply src/ota.html:126]: Ah, highlighting does not work as in azure DevOps.
In the code part it's spelled differently: `<code>Update succeded</code> `

Comment (Dirnei) [reply src/ota.html]: ah thanks, changed it

**Pull request merged**